test/spec/support/jasmine.json

```diff
@@ -0,0 +1,11 @@
+{
+  "spec_dir": "spec",
```

Comment:
Done.
It was in "spec" because that was the default location, but I agree that using "unittests" is a better name given that we also have integration tests here.